printrun/gui/__init__.py

```diff
@@ -255,8 +256,7 @@ def createGui(self, compact = False, mini = False):
             logpanel = self.newPanel(left_real_panel)
         viz_pane = VizPane(self, vizpanel)
         # Custom buttons
-        if wx.VERSION > (2, 9): self.cbuttonssizer = wx.WrapSizer(wx.HORIZONTAL)
-        else: self.cbuttonssizer = wx.GridBagSizer()
+        self.cbuttonssizer = wx.WrapSizer(wx.HORIZONTAL)
```

Comment:
Hmm looking further down it's negated AGAIN here. So yeah, looks like we can remove the two lines that set cbuttonsizer before this one.